lib/python/vdm/server/voltdbserver.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, database_id):
         """
         self.database_id = database_id
 
-    def start_database(self, recover=False):
+    def start_database(self, recover=False, blocking=-1):
```

Comment:
This input parameter, blocking, in start_database is not used, correct? If so, remove it.